### PR TITLE
Automation tests for settings page

### PR DESCRIPTION
**Pull request by lydia-karungi**
# :books: Linear Ticket :books:
- User should able to send and receive funds of same network eg BTC to BTC1
- Disabling any account reduces the total number of assets

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -121,13 +121,13 @@ jobs:
...
-      - name: Regression tests against Mainnet (PROD Agent)
-        uses: beemi/puppeteer-headful@v1.0.11
-        env:
-          CI: 'true'
-          DISPLAY: :99
-          SEED_WORDS: ${{ secrets.TEST_NET_SEED_WORDS }}
-          TWENTY_FOUR_SEEDS: ${{ secrets.TWENTY_FOUR_SEEDS }}
-        with:
-          args: yarn test:mainnet:prodagent:release
+##      Run tests against Prod mainnet
```
Comment (beemi) [inline .github/workflows/release.yml:124]: why this is changed?

File: test/e2e/manage_accounts.spec.js [hunk 2]
```diff
@@ -199,4 +200,80 @@ describe('Manage Accounts-["MAINNET","PULL_REQUEST_TEST"]', async () => {
     // Check web3 status as connected
     await dappPage.waitForSelector('#web3-status-connected', { visible: true })
   })
+
+  it.only('BTC1-BTC2 - Verify send between newly created accounts on the same network', async () => {
```
Comment (beemi) [inline test/e2e/manage_accounts.spec.js:204]: .only

File: test/e2e/manage_accounts.spec.js [hunk 3]
```diff
@@ -199,4 +200,80 @@ describe('Manage Accounts-["MAINNET","PULL_REQUEST_TEST"]', async () => {
...
+      await page.waitForSelector('#send_button_confirm', { visible: true, timeout: 60000 })
+    } catch (e) {
+      if (e instanceof puppeteer.errors.TimeoutError) {
+        await page.$eval('#send_review_button', (el) => el.click())
+      }
+    }
+
+    // Click send button
+    await page.waitForSelector('#send_button_confirm')
+    await page.click('#send_button_confirm')
```
Comment (beemi) [inline test/e2e/manage_accounts.spec.js:255]: after send, can you asset BTC 2 account has funds

File: test/e2e/manage_accounts.spec.js [hunk 4]
```diff
@@ -199,4 +200,80 @@ describe('Manage Accounts-["MAINNET","PULL_REQUEST_TEST"]', async () => {
...
+  })
+
+  it('BTC- Verify disabling an account reduces total number of assets', async () => {
+    // get total number of assets
+    const assetsCount1 = await overviewPage.getTotalAssets(page)
```
Comment (beemi) [inline test/e2e/manage_accounts.spec.js:260]: rename variable name to `assetsBeforeDisableManageAssets
`

**Review by beemi — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (lydia-karungi) [reply .github/workflows/release.yml]: @beemi Resolved this, am not sure why it appeared changed

File: test/e2e/manage_accounts.spec.js (hunk 2 above)
Comment (lydia-karungi) [reply test/e2e/manage_accounts.spec.js]: I have removed this @beemi

File: test/e2e/manage_accounts.spec.js (hunk 3 above)
Comment (lydia-karungi) [reply test/e2e/manage_accounts.spec.js:255]: @beemi I have added a check for this

File: test/e2e/manage_accounts.spec.js (hunk 4 above)
Comment (lydia-karungi) [reply test/e2e/manage_accounts.spec.js]: @beemi I have adjusted this

**Review by beemi — APPROVED** [review]

**Pull request merged**